applications/SwimmingDEMApplication/tests/candelier_tests/ProjectParametersDEMWithHistory.json

```diff
@@ -277,6 +277,12 @@
         "RotationOption"                   : true,
         "CleanIndentationsOption"          : false,
         "RemoveBallsInEmbeddedOption"      : true,
+        
+        "strategy_parameters" : {
```

Comment:
Let me insist. I think we better arrive at a consensus with this before we merge.

Reply:
I forgot about this. Well, the first time I used the term "strategy_parameters" it was because it contained information for the C++ strategy. So the name was ok then, in my opinion.
If now we are using this key for other purposes, maybe the name is not so correct. Are we really passing information to the solver? Is the solver anything more than a wrapper of the strategy? Even if the name was wrong, shoud we change (again) the json file? Right now I am in favor of keeping the current name. We can add another field called "solver_parameters" though.

Reply:
The rest of Kratos use "solver_settings" and the word "strategy" does not appear at all. That would be a reason to change to "solver_settings".

Reply:
Just exposing my thoughts. Anyone else's thoughts?

Reply:
+1 for unification => `solver_settings`

Reply:
changes uploaded